src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -253,6 +254,7 @@ const MeasureGroups = () => {
             scoringUnit: "",
           },
         });
+        setActiveTab("populations");
```

Comment:
do you think we can move this out of if-else and have just one `setActiveTab`?

Reply:
Oh yeah, that is right. Thank you, I have made the changes and pushed it again